tests/lax_numpy_test.py

```diff
@@ -180,10 +180,10 @@ def op_record(name, nargs, dtypes, shapes, rng_factory, diff_modes,
               inexact=True),
     op_record("arctan2", 2, float_dtypes, all_shapes, jtu.rand_small, ["rev"],
               inexact=True),
-    op_record("arcsinh", 1, number_dtypes, all_shapes, jtu.rand_positive, ["rev"],
-              inexact=True),
-    op_record("arccosh", 1, number_dtypes, all_shapes, jtu.rand_positive, ["rev"],
-              inexact=True),
+    op_record("arcsinh", 1, number_dtypes, all_shapes, jtu.rand_default, ["rev"],
+              inexact=True, tolerance={np.complex64: 2E-6, np.complex128: 2E-14}),
+    op_record("arccosh", 1, number_dtypes, all_shapes, jtu.rand_default, ["rev"],
+              inexact=True, tolerance={np.complex64: 2E-2, np.complex128: 2E-12}),
     op_record("arctanh", 1, number_dtypes, all_shapes, jtu.rand_small, ["rev"],
               inexact=True, tolerance={np.float64: 1e-9}),
```

Comment:
Is `arctanh` free of that problem, or should we also perform a similar check?

Reply:
arctanh may have that problem, but `jtu.rand_small` already tests all phase angles in the complex plane. The issue with the arcsinh and arccosh tests was that they were only testing inputs with positive real and imaginary parts.

Reply:
And `rand_small` is more appropriate than `rand_default`, I think, because of the domain for which arctanh is defined for real inputs.